### PR TITLE
Reviewer opt-out action and mailers for review un-/assign and opt-out

### PR DESCRIPTION
**Pull request by jseraf**

### CODE REVIEW

File: app/mailers/application_mailer.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  # TODO: set per environment email
+  default from: 'no-reply@environment-specific-domain.com'
```
Comment (mattbaumann1) [inline app/mailers/application_mailer.rb]: Will this get replaced once we have it?

File: config/environments/development.rb [hunk 2]
```diff
@@ -65,16 +65,13 @@
...
-  # Mailer configurations for Devise
+  config.action_mailer.smtp_settings = { address: '127.0.0.1', port: 1025 }
+  config.action_mailer.raise_delivery_errors = true
   config.subdomain = ''
-  config.app_domain = 'localhost:3000'
-  # config.action_mailer.delivery_method = :smtp
-  config.action_mailer.default_url_options = { host: config.app_domain, port: 3000 }
-  # config.action_mailer.perform_deliveries = true
+  config.app_domain = 'localhost'
+  config.action_mailer.default_url_options = { host: config.app_domain, port: 5000 }
```
Comment (mattbaumann1) [inline config/environments/development.rb]: Is this what will need to be changed?

File: app/mailers/application_mailer.rb (hunk 1 above)
Comment (jseraf) [reply app/mailers/application_mailer.rb]: yes. will include a note in #147

File: config/environments/development.rb (hunk 2 above)
Comment (jseraf) [reply config/environments/development.rb]: yes. will include a note in #147

**Pull request merged**